src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -6624,14 +6624,24 @@ private bool HasTopLevelNullabilityConversion(TypeWithAnnotations source, TypeWi
         }
 
         /// <summary>
-        /// Apply the conversion to the type of the operand and return the resulting type. (If the
-        /// operand does not have an explicit type, the operand expression is used for the type.)
-        /// If `checkConversion` is set, the incoming conversion is assumed to be from binding and will be
-        /// re-calculated, this time considering nullability. (Note that the conversion calculation considers
-        /// nested nullability only. The caller is responsible for checking the top-level nullability of
-        /// the type returned by this method.) `trackMembers` should be set if the nullability of any
-        /// members of the operand should be copied to the converted result when possible.
+        /// Apply the conversion to the type of the operand and return the resulting type.
+        /// If the operand does not have an explicit type, the operand expression is used for the type.
```

Comment:
>the operand expression is used for the type. [](http://example.com/codeflow?start=59&length=44)

Consider "the operand expression is used for the type." -> "the operand expression is used."

Reply:
```suggestion
        /// If the operand does not have an explicit type, the operand expression is used.
```